src/api/DebtIssuanceAPI.ts

```diff
@@ -216,7 +216,13 @@ export default class DebtIssuanceAPI {
     quantity: BigNumber,
     isIssue: boolean,
     callerAddress: Address = undefined,
-  ): Promise<(BigNumber)[][]> {
+  ): Promise<
+      [BigNumber, BigNumber, BigNumber] & {
+        totalQuantity: BigNumber;
+        managerFee: BigNumber;
+        protocolFee: BigNumber;
+      }
+    > {
```

Comment:
nice tuple type! Since this is public API, I'm sure consumers will appreciate this more precise type definition